features/support/env.rb

```diff
@@ -42,6 +42,15 @@
   log.error(message + "failed")  if scenario.failed?
 end
 
+After('@users') do
+  user_name = config["features"]["users"]["user_name"]
+  users = control_node.openstack.user.list
+  user_names = users.map(&:name)
+  if user_names.include? user_name
+    control_node.openstack.user.delete user_name
+  end
+end
+
```

Comment:
> This could be done more transparently in Background with And the test user does not exists" right after the first Given.

But wouldn't that mean that the cleaning-up part is not executed _after_ all tests are finished. So test_user would still exist then and only get deleted at a rerun?